docs/examples/hypervisors/kvm/modules/kvm-hypervisor/README.md

```diff
@@ -0,0 +1,116 @@
+KVM Installation module
+=======================
+
+This Module is responsible for setting up a custom KVM based Virtual Machines on top of a KVM hypervisor on Oracle Cloud Infrastructure. KVM domain is installed as part of this module.
+
+ #Oracle Linux image
+
+###Module required Variables
```

Comment:
Formatting is not right for much of this page - try the "view" button and take a look.